src/nouislider.js

```diff
@@ -226,10 +224,4 @@ function init(Survey) {
     },
   };
   Survey.CustomWidgetCollection.Instance.addCustomWidget(widget, "customtype");
-}
-
-if (typeof Survey !== "undefined") {
```

Comment:
If we remove this call, custom widget will not be registered in non-modular applications